### PR TITLE
October 2023

### PR DESCRIPTION
**Pull request by m-kauppinen**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [globalization/locale/dictionaries-spelling.md](https://github.com/MicrosoftDocs/globalization/blob/9c86e20c0fedb503edbae2c91997ff7dad50660b/globalization%2Flocale%2Fdictionaries-spelling.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/19f4b8d0-f984-4c02-abfe-d2adbd63f693) | [:white_check_mark:](# "Successfully processed") |
| [globalization/locale/user-preferences.md](https://github.com/MicrosoftDocs/globalization/blob/9c86e20c0fedb503edbae2c91997ff7dad50660b/globalization%2Flocale%2Fuser-preferences.md) | 92 | 100 | 87 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e7071a28-efad-409d-8a4c-ce4090e87180) | [:white_check_mark:](# "Successfully processed") |
| [globalization/methodology/how-to-release-products-internationally.md](https://github.com/MicrosoftDocs/globalization/blob/9c86e20c0fedb503edbae2c91997ff7dad50660b/globalization%2Fmethodology%2Fhow-to-release-products-internationally.md) | 91 | 100 | 87 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d7b51ac9-9f1a-40ca-9d19-7c4079287bdf) | [:white_check_mark:](# "Successfully processed") |
| [globalization/toc.yml](https://github.com/MicrosoftDocs/globalization/blob/9c86e20c0fedb503edbae2c91997ff7dad50660b/globalization%2Ftoc.yml) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/67bf2894-6843-43a1-8836-b54db8d801f9) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinx-at-msft2** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [globalization/locale/dictionaries-spelling.md](https://github.com/MicrosoftDocs/globalization/blob/6233725501754d9acbdd8139d0b3247de9bd68d5/globalization%2Flocale%2Fdictionaries-spelling.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ba7f85ee-9a4f-4677-8669-2ee523a1cd4c) | [:white_check_mark:](# "Successfully processed") |
| [globalization/locale/user-preferences.md](https://github.com/MicrosoftDocs/globalization/blob/6233725501754d9acbdd8139d0b3247de9bd68d5/globalization%2Flocale%2Fuser-preferences.md) | 92 | 100 | 87 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/37cc6612-e736-4d6f-8eed-728b55b89fed) | [:white_check_mark:](# "Successfully processed") |
| [globalization/methodology/how-to-release-products-internationally.md](https://github.com/MicrosoftDocs/globalization/blob/6233725501754d9acbdd8139d0b3247de9bd68d5/globalization%2Fmethodology%2Fhow-to-release-products-internationally.md) | 91 | 100 | 87 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/692cbd97-721e-4e03-af57-c71b6f791c3d) | [:white_check_mark:](# "Successfully processed") |
| [globalization/toc.yml](https://github.com/MicrosoftDocs/globalization/blob/6233725501754d9acbdd8139d0b3247de9bd68d5/globalization%2Ftoc.yml) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dd5ece07-ced9-41d0-b31f-c6571725dfab) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: globalization/locale/user-preferences.md [hunk 1]
```diff
@@ -1,6 +1,41 @@
...
 # Identifying user preferences
+
+Products intended for international markets need to be able to identify and follow users’ language and country preferences. Windows has a dedicated group of settings for this purpose under **Settings** > **Time & language** > **Language & region**. They consist of:
+
+- **Windows display language**: used for the Windows UI and for features such as File Explorer
+- **Preferred languages**: a Microsoft Store application uses the first language in this list that is supported by the app
+- **Country or region**: Windows and applications can use this setting to customize the content that is presented to the user
+- **Regional format**: Windows and apps use this setting to format dates and times according to the conventions of the selected locale
+
+All Windows language settings are available via an API.
```
Comment (jowilco) [inline globalization/locale/user-preferences.md:19]: Link to /dotnet/api/system.globalization.cultureinfo

File: globalization/methodology/how-to-release-products-internationally.md [hunk 2]
```diff
@@ -4,8 +4,97 @@ description: Introduction to strategy decisions and other important concerns for
...
+
+The pricing of your product might also need adjustment for the target market. You don’t want to underprice or overprice your offering. You might want to consider discounts in the beginning to help grow your user base in the target market. But be mindful of arbitrage. If you offer a cheaper price in the target market, your source market customer might take advantage of that and buy your product there. The more expensive your product is, the more important it's to consider this possibility. For example, industrial products can cost millions of dollars.
+
+### Localization
+
+Your product should be designed according to the principles of internationalization, making it as easy to localize as possible. Thus it should have support for [locales](../locale/locale.md) and [language settings](../locale/language-settings.md), and its localizable resources should be [externalized](../localization/localize-software.md). Besides the product itself, you should have selected a localization partner and agreed on a process with them. For more information about the whole localization process, refer to [Localizing your product](../localization/localization-overview.md).
```
Comment (jowilco) [inline globalization/methodology/how-to-release-products-internationally.md:86]: Let's also add the internationalization link here.

File: globalization/locale/user-preferences.md [hunk 3]
```diff
@@ -1,6 +1,41 @@
 ---
 title: Identifying user preferences
-description: International users are mobile and may use more than one language, so it's important to identify the user's preferences for user interface, input, and content.
-ms.date: 01/01/2022
+description: International users are mobile and might use more than one language, so it's important to identify the user's preferences for user interface, input, and content.
```
Comment (brunolewin-msft) [inline globalization/locale/user-preferences.md:3]: The article explains **How to identify preferences of international users for user interface, input, and content, including language and regional formats** rather than the **mobile/multilingual user** aspect, so maybe the former could work better as description? (thinking mostly from a Search results POV)

File: globalization/methodology/how-to-release-products-internationally.md [hunk 4]
```diff
@@ -4,8 +4,97 @@ description: Introduction to strategy decisions and other important concerns for
...
+
+### Product delivery
+
+Physical products need shipping to local warehouses and distribution deals to stores. Customers are going to be unhappy if they can only get your product by waiting many weeks for expensive international delivery.
+
+If your product is something that could be manufactured in-country, that is worth considering as it could let you avoid many import-related issues. These include customs duties and tariffs, taxes, import licensing, and shipping, for example.
+
+### Support
```
Comment (brunolewin-msft) [inline globalization/methodology/how-to-release-products-internationally.md:94]: "Local support and infrastructure"?

File: globalization/locale/user-preferences.md (hunk 3 above)
Comment (m-kauppinen) [reply globalization/locale/user-preferences.md:3]: How about "International users might use more than one language with their operating system and apps, so it's important to identify the user's preferences for user interface, input, and content, including language and regional preferences."?

Comment (brunolewin-msft) [reply globalization/locale/user-preferences.md:3]: Sorry I was not clear - my concern was that the most important of your description would get truncated in Search results, so was suggesting we shorten to "How to identify preferences of international users for user interface, input, and content, including language and regional formats" so the essential part is visible in the search results.

Comment (m-kauppinen) [reply globalization/locale/user-preferences.md:3]: Right! I changed the description to that.

File: globalization/methodology/how-to-release-products-internationally.md (hunk 2 above)
Comment (m-kauppinen) [reply globalization/methodology/how-to-release-products-internationally.md:86]: Done.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [globalization/locale/dictionaries-spelling.md](https://github.com/MicrosoftDocs/globalization/blob/423d7b23bd394613bc7749923ce7d6f747c890c9/globalization%2Flocale%2Fdictionaries-spelling.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5c81efae-109c-434b-a6cc-91c9b09b5550) | [:white_check_mark:](# "Successfully processed") |
| [globalization/locale/user-preferences.md](https://github.com/MicrosoftDocs/globalization/blob/423d7b23bd394613bc7749923ce7d6f747c890c9/globalization%2Flocale%2Fuser-preferences.md) | 89 | 100 | 87 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9d664df0-b8ea-4208-a3f2-a5ae54af2576) | [:white_check_mark:](# "Successfully processed") |
| [globalization/methodology/how-to-release-products-internationally.md](https://github.com/MicrosoftDocs/globalization/blob/423d7b23bd394613bc7749923ce7d6f747c890c9/globalization%2Fmethodology%2Fhow-to-release-products-internationally.md) | 87 | 100 | 87 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5d8a8894-df47-4463-ba3f-7b53852fb19d) | [:white_check_mark:](# "Successfully processed") |
| [globalization/toc.yml](https://github.com/MicrosoftDocs/globalization/blob/423d7b23bd394613bc7749923ce7d6f747c890c9/globalization%2Ftoc.yml) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2a14e582-cc10-4a76-ac1d-2e935fa6f8c0) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: globalization/locale/user-preferences.md (hunk 1 above)
Comment (m-kauppinen) [reply globalization/locale/user-preferences.md:19]: Done.

**Review by brunolewin-msft — APPROVED** [review]

File: globalization/locale/user-preferences.md (hunk 1 above)
Comment (jowilco) [reply globalization/locale/user-preferences.md:19]: Unfortunately, link applied in the wrong place.
How about rewording to:
All Windows language settings are available via the [.NET API's CultureInfo Class](/dotnet/api/system.globalization.cultureinfo)

File: globalization/methodology/how-to-release-products-internationally.md (hunk 2 above)
Comment (jowilco) [reply globalization/methodology/how-to-release-products-internationally.md]: I see the change, but again not quite correct. Sorry if I wasn't clear.
Please remove the new "For more information about globalization, refer to [System.Globalization Namespace](/dotnet/api/system.globalization?view=net-7.0)." and change "Your product should be designed according to the principles of internationalization..." to "Your product should be designed according to the principles of [internationalization](software-internationalization.md)..."

File: globalization/locale/user-preferences.md (hunk 1 above)
Comment (jowilco) [reply globalization/locale/user-preferences.md:19]: Also, note that I don't have the ?view=net-7.0. That will restrict the link to the named version of .NET, but we'll want it to go to the latest.

Comment (brunolewin-msft) [reply globalization/locale/user-preferences.md:19]: @jowilco - this conversation makes me wonder if we should be _even more explicit_ and take the reader straight to the section that explain usage of CurrentCulture and CurrentUICulture - e.g. 
Language and regional format settings are available via .NET API's CultureInfo Class (dotnet/api/system.globalization.cultureinfo**#the-current-culture-and-current-ui-culture**)

Comment (jowilco) [reply globalization/locale/user-preferences.md:19]: @brunolewin-msft @m-kauppinen While I'm not opposed to using the # / heading, in this case the reader is taken to a heading quite a way down the document, missing out on some very useful information.

Comment (brunolewin-msft) [reply globalization/locale/user-preferences.md:19]: Agree. Note the idea here was to get to them to a place where they can immediately see the relationship between the two Windows settings and relevant CultureInfo properties (and once they know that, they can read the 10 pages of more general info about .Net culture).

Comment (m-kauppinen) [reply globalization/locale/user-preferences.md:19]: > Also, note that I don't have the ?view=net-7.0. That will restrict the link to the named version of .NET, but we'll want it to go to the latest.

For some reason my browser had gone to the 7.0 page when I tested the link. Will take it out.

File: globalization/methodology/how-to-release-products-internationally.md (hunk 2 above)
Comment (m-kauppinen) [reply globalization/methodology/how-to-release-products-internationally.md]: Fixed.

File: globalization/locale/user-preferences.md (hunk 1 above)
Comment (m-kauppinen) [reply globalization/locale/user-preferences.md:19]: And fixed.

**Review by jowilco — APPROVED** [review]
